initrd/bin/kexec-boot

```diff
@@ -1,13 +1,89 @@
 #!/bin/sh
 # Launches kexec from saved configuration entries
-set -e -o pipefail
-. /tmp/config
-. /etc/functions
+
+usage() {
+	cat <<End-of-message
+Usage
+=====
+
+Launches kexec from saved configuration entries
+	
+$0 -b /boot -e 'kexec params|...|...' [<options>]
+	
+Options
+-------
+
+### -a <additional confiurations>
+
+Anything provided to the -a argument is appended to the kernel command line (usually comes from parsing the grub, syslinux, BLS files). Ex.
+
+	-a 'intel_iommu=on intel_iommu=igfx_off'
+
```

Comment:
@jtmoree-github-com 
Those normally come from [board configuration statements](https://github.com/osresearch/heads/blob/0d5b3d75e7b84ca85ff252fe5fe23aea990da435/initrd/bin/kexec-boot#L32-L33)

Reply:
I have reworded this to follow the suggestion below